### PR TITLE
Update the readme to explain the context for the PR

### PR DESCRIPTION
**Pull request by spbrogan**

### CODE REVIEW

**Comment by spbrogan** [comment]
This PR process is great.  If only we could move edk2 to a similar process.  Think of the hours it would save.  Maybe someone could actually get a few bug fixes in.

File: README.md [hunk 1]
```diff
@@ -54,3 +54,7 @@ Initial release of ...
 * [Markdown Syntax Reference](https://help.github.com/articles/markdown-basics/)
 
 **Enjoy!**
+
+Don't complete this PR.  I just want to see what it looks like via the email reflector.
```
Comment (spbrogan) [inline README.md]: I wanted to add a single comment without a review but this comment should also have context of the line i added the comment to.  Lets see.

File: README.md [hunk 2]
```diff
@@ -54,3 +54,7 @@ Initial release of ...
 * [Markdown Syntax Reference](https://help.github.com/articles/markdown-basics/)
 
 **Enjoy!**
```
Comment (spbrogan) [inline README.md:56]: Do you really think this is enjoyable?  :)  This is a code review let see what happens.

**Review by spbrogan — COMMENTED** [review]
I can't submit a review approval since it was my change.  I can however add comments for a review.

**Comment by spbrogan** [comment]
Lets add another general comment to make sure the app is still running.

**Comment by spbrogan** [comment]
what happens if i put in some advanced characters.  @spbrogan 

- [x] task 1
- [x]  task 2

File: README.md (hunk 2 above)
Comment (matthewfcarlson) [reply README.md:56]: I don't know what happens here

File: README.md (hunk 1 above)
Comment (matthewfcarlson) [reply README.md]: Wondering how this makes sense. :)

**Comment by matthewfcarlson** [comment]
Hello there!

**Comment by matthewfcarlson** [comment]
Here we go

**Comment by matthewfcarlson** [comment]
Another test?

**Comment by matthewfcarlson** [comment]
here's one more

**Comment by matthewfcarlson** [comment]
here's johnny

**Comment by matthewfcarlson** [comment]
And his less known but equally unhinged cousin, Yonnie

**Comment by matthewfcarlson** [comment]
Yawnie?

**Comment by matthewfcarlson** [comment]
Yanni?

**Comment by matthewfcarlson** [comment]
yanny?

**Comment by matthewfcarlson** [comment]
What's up doc?

**Comment by matthewfcarlson** [comment]
Not much. How about you? I'm doing great really.

**Pull request closed without merging**